src/components/Filters/index.tsx

```diff
@@ -286,43 +284,66 @@ class Filters extends React.Component<FiltersProps, FiltersState> {
 		}, () => this.props.onFiltersUpdate && this.props.onFiltersUpdate(this.state.filters));
 	}
 
+	shouldRenderComponent(mode: FilterRenderMode): boolean {
+		// If a render mode is not specified, render all components
+		if (!this.props.renderMode) {
+			return true;
+		}
+
+		const allowedModes = Array.isArray(this.props.renderMode) ? this.props.renderMode : [ this.props.renderMode ];
+
+		if (allowedModes.indexOf('all') > -1) {
```

Comment:
How come we're not using `.includes('all')`?

Reply:
I'll change it to use includes